cli/src/command/account.rs

```diff
@@ -126,11 +126,14 @@ pub enum AccountCommand {
     /// List the unspent outputs.
     UnspentOutputs,
     /// Cast given votes for a given event
-    Vote { event_id: String, answers: Vec<u8> },
+    Vote {
+        event_id: ParticipationEventId,
+        answers: Vec<u8>,
+    },
     /// Stop participating to a given event
-    StopParticipating { event_id: String },
+    StopParticipating { event_id: ParticipationEventId },
     /// Calculate the participation overview of the account
-    ParticipationOverview,
+    ParticipationOverview { event_id: Option<ParticipationEventId> },
```

Comment:
I think you can make it a Vec and clap should be able to deal with it...

Reply:
Hm how to provide it then? Multiple events fail for me
```
participation overview 0x0344c97dc9cddc47f880fc1934e361636bf83029268f17faaac97c7be3865f7f 0x013d2ca528a25124231c720095859802f354f8760c0ff1301daba052d4419cf5
error: unrecognized subcommand
```

Reply:
Maybe it doesn't like `Option<Vec<_>>`? I would argue that if we have a Vec, the Option is not needed in the CLI anyway

Reply:
Assuming clap accepts empty vecs ofc

Reply:
Yeah I tried first `Option<Vec<_>>`, but that didn't event accept a single one and then I tried `Vec<_>` and this seems only to work if I provide a single one
But usually one only cares about a single one anyways, no?

Reply:
Ok got it working, was missing some clap arguments